src/js/SearchBar.js

```diff
@@ -0,0 +1,22 @@
+(function() {
+
+	$('.SearchBar').each( function() {
```

Comment:
This would require jQuery to be bundled with the pattern library. The JS itself is pretty simple, can we just use vanilla?